.github/actions/deploy/deploy.py

```
@@ -0,0 +1,589 @@
+                setattr(self.args, arg_name, self.str_to_bool(current_value))
+
+    def setup_environment(self):
+        """Extract repository information and set up environment."""
+        print('🔧 Setting up deployment environment...')
+
+        # Extract repo owner from github repository
+        if self.args.github_repository:
+            self.repo_owner = self.args.github_repository.split('/')[0]
+            print(f'📂 Detected repository owner: {self.repo_owner}')
+        else:
+            raise ValueError('GitHub repository not provided')
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)